www-servers/nginx-unit/nginx-unit-1.3-r1.ebuild

```diff
@@ -4,6 +4,10 @@
 EAPI=7
 
 MY_P="unit-${PV}"
+MY_IUSE="perl python ruby"
```

Comment:
You don't need those vars before inherit. Please try to keep them near their first use, otherwise it's hard to find them; i.e. MY_P above DESCRIPTION, MY_IUSE above IUSE.